### ISSUE TITLE
Planning-summary page-"since" date not populating on all applications - 14566021

### ISSUE THREAD

**Issue by MLWilkins** [dhenry-KCI/FredCo-Post-Go-Live-#552]
The date field under "since" is populating on some APs but not others.  It is random across all Planning AP types (example, some site plans have it and others do not). Also seems random on which milestone its in.

![image](https://user-images.githubusercontent.com/50637067/81432520-10d45f80-9131-11ea-9118-32ad64afd44e.png)

![image](https://user-images.githubusercontent.com/50637067/81432634-47aa7580-9131-11ea-821c-daf3c7a4f308.png)

![image](https://user-images.githubusercontent.com/50637067/81432709-6b6dbb80-9131-11ea-8f60-ea9872f0075e.png)

**Comment by megisys** [comment]
Ticket - 14566021

**Comment by MLWilkins** [comment]
@megisys  @JuanitaMiles 

I noticed the AP used as an example above now has the date populated.  When the ticket was created I had found multiple examples.  This morning I noticed the one I took a pic of above was working and checked a few dozen other APs of various types and at various milestones and all appear to work.  Its ok by me if the extreme ticket is canceled and this ticket is closed. Maybe the issue was related to the cache issue that caused the map issue?
